quickjspp.hpp

```diff
@@ -1694,6 +1714,14 @@ class Context
         assert(ptr);
         return *static_cast<Context *>(ptr);
     }
+
+    static std::string readFile(const char * filename)
+    {
+        std::ifstream f(filename, std::ios::in | std::ios::binary);
```

Comment:
Should probably be a free function since it doesn't have anything to do with Context.
Also it returns empty string if the file doesn't exist